api/src/main/java/org/openmrs/module/fhir2/apiext/dao/impl/FhirMedicationAdministrationPerformerDaoImpl.java

```diff
@@ -0,0 +1,10 @@
+package org.openmrs.module.fhir2.apiext.dao.impl;
+
+import org.openmrs.module.fhir2.api.dao.impl.BaseFhirDao;
+import org.openmrs.module.fhir2.apiext.dao.FhirMedicationAdministrationPerformerDao;
+import org.openmrs.module.ipd.api.model.MedicationAdministrationPerformer;
+import org.springframework.stereotype.Component;
+
+@Component
+public class FhirMedicationAdministrationPerformerDaoImpl extends BaseFhirDao<MedicationAdministrationPerformer> implements FhirMedicationAdministrationPerformerDao {
+}
```

Comment:
same as above comment

Reply:
@kavitha-sundararajan Added Implementation which will return super.get(uuid) ! Will not be able to remove this Impl as we will not able to autowire it without this https://github.com/component